code/game/objects/items/weapons/powerfist.dm

```diff
@@ -89,7 +90,7 @@
 	if(!tank)
 		to_chat(user, "<span class='warning'>[src] can't operate without a source of gas!</span>")
 		return
-	if(tank && !tank.air_contents.boolean_remove(gasperfist * fisto_setting))
+	if(tank && !tank.air_contents.boolean_remove(((gasperfist * fisto_setting) * tank.air_contents.return_volume()) / (8.3145 * tank.air_contents.return_temperature())))
```

Comment:
`8.3145` should be converted into a define with documentation as to how that value was reached